python/rascaline/examples/long-range-descriptor.py

```diff
@@ -0,0 +1,365 @@
+"""
+.. _userdoc-tutorials-long-range-descriptor:
+
+Pure LODE long-range descriptor
+===============================
+
+.. start-body
+
+We start the example by loading the required packages
+"""
+
+# %%
+
+import ase
+import ase.visualize.plot
+import matplotlib.pyplot as plt
+import numpy as np
+from ase.build import molecule
+from metatensor import TensorMap
+
+from rascaline import LodeSphericalExpansion, SphericalExpansion
+from rascaline.utils import LodeDensity, LodeSpliner, MonomialBasis, SoapSpliner
+
+
+# %%
+#
+# **Single water molecule (short range) system**
+#
+# Our first test system is a single water molecule with a :math:`15\,\mathrm{Å}` vacuum
+# layer around it.
+
+
+atoms = molecule("H2O", vacuum=15, pbc=True)
+
+# %%
+# We choose a ``cutoff`` for the projection of the spherical expansion and the neighbor
+# search of the real space spherical expansion.
+
+cutoff = 3
+
+# %%
+# We visualize the system and draw a gray circle with the radius of the cutoff.
+
+fig, ax = plt.subplots()
+
+ase.visualize.plot.plot_atoms(atoms)
+
+cutoff_circle = plt.Circle(
+    xy=atoms[0].position[:2],
+    radius=cutoff,
+    color="gray",
+    ls="dashed",
+    fill=False,
+)
+ax.add_patch(cutoff_circle)
+
+ax.set_xlabel("Å")
+ax.set_ylabel("Å")
+
+fig.show()
+
+
+# %%
+#
+# As you can see, for a single water molecule, the cutoff includes all atoms of the
+# system. The combination of the test system and the ``cutoff`` aims to demonstrate that
+# the full atomic fingerprint is contained within the ``cutoff``. By later subtracting
+# the LODE density and the short-range density, we will observe that the difference
+# between them is zero, indicating that this represents the short-range system.
+#
+# Furthermore, we choose a high potential exponent to emulate the rapidly decaying LODE
+# density and mimic the polar-polar interactions of water.
+
+
+potential_exponent = 3
+
+
+# %%
+# We now define some typical hyperparameters to compute the spherical expansions.
+
+max_radial = 5
+max_angular = 1
+atomic_gaussian_width = 1.2
+center_atom_weight = 1.0
+
+
+# %%
+# We choose a relatively low spline accuracy (default is ``1e-8``) to achieve quick
+# computation of the spline points. You can increase the spline accuracy if required,
+# but be aware that the time to compute these points will increase significantly!
+
+
+spline_accuracy = 1e-2
+
+
+# %%
+# As a projection basis, we don't use the usual :py:class:`GtoBasis
+# <rascaline.utils.GtoBasis>` which is commonly used for short range descriptors.
+# Instead, we select the :py:class:`MonomialBasis <rascaline.utils.MonomialBasis>` which
+# is the optimal basis for the LODE descriptor.
+
+
+basis = MonomialBasis(cutoff=cutoff)
+
+
+# %%
+# For the density, we choose a smeared power law as used in LODE, which does not decay
+# exponentially like a :py:class:`Gaussian density <rascaline.utils.GaussianDensity>`
+# and is therefore suited to describe long-range interactions between atoms.
+
```

Comment:
might be nice to plot this density & the Gaussian density to make this point clearer

Reply:
Yep! A log-log plot should now be in.